packages/inference/test/HfInference.spec.ts

```diff
@@ -1175,4 +1175,76 @@ describe.concurrent("HfInference", () => {
 		},
 		TIMEOUT
 	);
+
+	describe.concurrent(
+		"Hyperbolic",
+		() => {
+			const client = new HfInference(env.HF_HYPERBOLIC_KEY);
+
+			HARDCODED_MODEL_ID_MAPPING.hyperbolic = {
+				"meta-llama/Llama-3.2-3B-Instruct": "meta-llama/Llama-3.2-3B-Instruct",
+				"meta-llama/Llama-3.3-70B-Instruct": "meta-llama/Llama-3.3-70B-Instruct",
+				"stabilityai/stable-diffusion-2": "stabilityai/stable-diffusion-2",
+				"meta-llama/Llama-3.1-405B-BASE-FP8": "meta-llama/Llama-3.1-405B-BASE-FP8",
```

Comment:
Note that the keys should be HF model ids (it's not the case for the last one at least)